docs/host-list.rst

```diff
@@ -0,0 +1,81 @@
+.. source: https://gist.github.com/Twentysix26/cb4401c6e507782aa6698e9e470243ed
+
+.. _host-list:
+
+=============
+VPS providers
+=============
+
+.. note::
+    This doc is written for the :ref:`hosting section <getting-started-hosting>`
+    of the :ref:`getting started <getting-started>` guide. Please take a look
+    if you don't know how to host Red on a VPS.
+
+This is a list of the recommanded VPS providers.
+
+.. warning::
+    Please be aware that a Linux server is controlled through command line.
+    If you don't know Unix basics, please take a look at `this guide
+    <https://www.digitalocean.com/community/tutorials/an-introduction-to-linux-basics>`_
+    from DigitalOcean which will introduct you to the Linux basics.
+
+-------------
+Linux hosting
+-------------
+
++------------------------------------+------------------------------------------------------+
+|Link                                |Description                                           |
++====================================+======================================================+
+|`Scaleway                           |Incredibly cheap but powerful VPSes, owned by         |
+|<https://www.scaleway.com/>`_       |`<https://online.net/>`_, based in Europe.            |
++------------------------------------+------------------------------------------------------+
+|`DigitalOcean                       |US-based cheap VPSes. The gold standard. Locations    |
+|<https://m.do.co/c/71288f6af536/>`_ |available world wide.                                 |
```

Comment:
I don't mind if we want to offer a referral link, but we should be explicit that it is one if we are going to do this.

Reply:
Whose referral link is this? @retke's? The recipient of the benefits going to the link owner should only ever be the Cog Creators organisation... I understand he's putting a lot of work into these docs but putting a link in there that makes the contributor money is unfair and against the spirit of open source development...

Reply:
For the record, this is the DO referral link we have as a CC on the discord server. ( I went to compare it when I saw, and my response would have been different had it not been)

Reply:
Right. Do we know who it belongs to?

Reply:
Pull discussion back to discord on this, but for now, the referral link should be removed.